### PR TITLE
[PBE-000] add connection regions

### PR DESCRIPTION
**Pull request by xernobyl**

### CODE REVIEW

File: src/StreamClient.cs [hunk 1]
```diff
@@ -168,14 +168,20 @@ private static string GetRegion(StreamApiLocation location)
...
+                case StreamApiLocation.Mumbai:
+                    return "mumbai";
                 case StreamApiLocation.Singapore:
                     return "singapore";
                 case StreamApiLocation.Sidney:
                     return "sydney";
+                case StreamApiLocation.Oregon:
+                    return "oregon";
+                case StreamApiLocation.Ohio:
+                    return "Ohio";
```
Comment (JimmyPettersson85) [inline src/StreamClient.cs:184]: nitpick, but maybe lowercase here to be consistent? :smile:

Comment (xernobyl) [reply src/StreamClient.cs]: Missed that one.

**Review by JimmyPettersson85 — APPROVED** [review]

**Comment by Miles-Yang** [comment]
I want to know when you're going to release added regions, and push to nuget? Now stream-net in nuget is 6.7.0, and I saw your 6.8.0 release is not included this. I need to use ohio.

**Comment by JimmyPettersson85** [comment]
@xernobyl Can you create a new release please? Seems like the one created for this PR failed https://github.com/GetStream/stream-net/actions/runs/7755466682

**Comment by Miles-Yang** [comment]
when release success. How can I get the latest nuget version? I saw 6.8.0 is released, but nuget's latest version is still 6.7.0. I want to get a version include this add regions.

**Comment by xernobyl** [comment]
Hello, we had a problem in the CI that should be fixed now: https://github.com/GetStream/stream-net/releases/tag/6.8.1

Also available on nuget: https://www.nuget.org/packages/stream-net/

**Comment by Miles-Yang** [comment]
oh, that's really cool. Thank you guys, make me fix the issue so fast.

**Pull request merged**